wolfcrypt/src/port/atmel/atmel.c

```diff
@@ -896,12 +909,80 @@ int atcatls_verify_signature_cb(WOLFSSL* ssl, const byte* sig, unsigned int sigS
     return ret;
 }
 
+static int atcatls_set_certificates(WOLFSSL_CTX *ctx) 
+{
+    int ret = 0;
+    ATCA_STATUS status;
+    size_t signerCertSize = 0;
+    uint8_t signerCert[signerCertSize];
+    size_t deviceCertSize = 0;
+    uint8_t deviceCert[deviceCertSize];
```

Comment:
Same here... `deviceCertSize` is 0 zero. This is usually `552` for `tng_atcacert_read_device_cert`.